src/cmdstan/arguments/arg_log_prob_constrained_params.hpp

```diff
@@ -6,9 +6,8 @@
 namespace cmdstan {
 /**
  * Argument for providing a file of parameters on the constrained scale
- * for use with the 'log_prob' method. The file can be in CSV or JSON format
- * and should contain a variable 'params_r' with either a vector or list/array
- * of vectors of constrained parameter values.
+ * for use with the 'log_prob' method. The file can be in JSON or R Dump format,
```

Comment:
Do they need to be named with the `.json` and `.R` convention?  If so, please doc.  If not, please explain how it knows which is which (can just say "format is autodetected" or something if that's the case).